packages/wrangler/src/api/dev.ts

```diff
@@ -1,9 +1,9 @@
-import { fetch } from "undici";
+import { fetch, Request } from "undici";
```

Comment:
dumb question, should this be imported as a type?

Reply:
Indeed it should, not a dumb question at all